src/node_http_common.h

```diff
@@ -24,61 +24,62 @@ class Environment;
+  V(CONTENT_TYPE, "content-type")                                              \
+  V(COOKIE, "cookie")                                                          \
+  V(DATE, "date")                                                              \
+  V(ETAG, "etag")                                                              \
+  V(FORWARDED, "forwarded")                                                    \
+  V(HOST, "host")                                                              \
+  V(IF_MODIFIED_SINCE, "if-modified-since")                                    \
+  V(IF_NONE_MATCH, "if-none-match")                                            \
+  V(IF_RANGE, "if-range")                                                      \
+  V(LAST_MODIFIED, "last-modified")                                            \
+  V(LINK, "link")                                                              \
+  V(LOCATION, "location")                                                      \
+  V(RANGE, "range")                                                            \
+  V(REFERER, "referer")                                                        \
+  V(SERVER, "server")                                                          \
+  V(SET_COOKIE, "set-cookie")                                                  \
+  V(STRICT_TRANSPORT_SECURITY, "strict-transport-security")                    \
+  V(TRANSFER_ENCODING, "transfer-encoding")                                    \
+  V(TE, "te")                                                                  \
+  V(UPGRADE_INSECURE_REQUESTS, "upgrade-insecure-requests")                    \
+  V(UPGRADE, "upgrade")                                                        \
+  V(USER_AGENT, "user-agent")                                                  \
+  V(VARY, "vary")                                                              \
+  V(X_CONTENT_TYPE_OPTIONS, "x-content-type-options")                          \
+  V(X_FRAME_OPTIONS, "x-frame-options")                                        \
+  V(KEEP_ALIVE, "keep-alive")                                                  \
+  V(PROXY_CONNECTION, "proxy-connection")                                      \
+  V(X_XSS_PROTECTION, "x-xss-protection")                                      \
+  V(ALT_SVC, "alt-svc")                                                        \
+  V(CONTENT_SECURITY_POLICY, "content-security-policy")                        \
+  V(EARLY_DATA, "early-data")                                                  \
+  V(EXPECT_CT, "expect-ct")                                                    \
+  V(ORIGIN, "origin")                                                          \
+  V(PURPOSE, "purpose")                                                        \
+  V(TIMING_ALLOW_ORIGIN, "timing-allow-origin")                                \
+  V(X_FORWARDED_FOR, "x-forwarded-for")                                        \
+  V(PRIORITY, "priority")
```

Comment:
the diff here is larger only because of `format-cpp`.... the only actual change is the addition of the `V(PRIORITY, "priority")` at the end of the list.